BankLibrary/PutAccountParameters.cs

```diff
@@ -0,0 +1,11 @@
+namespace BankLibrary
+{
+    public class PutAccountParameters
+    {
+        public int Id { get; set; }
+
+        public decimal Amount { get; set; }
+
+        public MoneyPutted MoneyPutted { get; set; }
```

Comment:
As I described, all Notifiers should go to CreateAccount params, we will do all subscriptions in one go